### ISSUE TITLE
Custom Coordinate Reference System not working at all

### ISSUE THREAD

**Issue by Meibes** [qgis/QGIS#32866]
TL;DR: deleting the profile and creating a new profile solved the issue

I was working on a project and added some shapefiles when QGIS told me it did not recognize the crs of some shapefiles, which was strange because a few days ago everything was fine. I tried to edit my Custom Coordinate Reference Systems but whenever I hit "OK" nothing did happen, the dialog stayed open. It only closed when I hit "OK" and the list was empty.

My setup: Windows 7 with 3.10 3.4.13 and 2.18.28 installed simultaneously via standalone installer. I tend to have some instances of the 3.10 and 3.4.13 open at the same time.

**Comment by roya0045** [comment]
Could you confirm what version of Proj you have.

**Comment by Meibes** [comment]
@roya0045 
PROJ | Rel. 5.2.0, September 15th, 2018, sorry for the late answer

could it be that the profile can get corrupted if you have like 5 instances of qgis open and close them all in a short time window?

**Comment by roya0045** [comment]
@Meibes I'm not too sure, but if I were you I would (maybe wait for the next point release? and ) update QGIS, it now comes with PROJ6 and that might also break your custom projections depending on how they are defined.
